From: Sebastien Guiriec <s-guiriec@ti.com>
Date: Sat, 22 Jan 2011 15:40:46 +0000 (+0100)
Subject: OMAP4: powerdomain: Fix ABE context lost offset
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=1ad8197aa77c83f1241c8ddd2af5a76857e945ab

OMAP4: powerdomain: Fix ABE context lost offset

Fix offset for ABE PRM register for context lost offset.

Change-Id: Ic1ed1335be20132efd0ada6ea0034f14d996ed28
Signed-off-by: Sebastien Guiriec <s-guiriec@ti.com>
---

Index: android-2.6.35/arch/arm/mach-omap2/powerdomains44xx.h
===================================================================
--- android-2.6.35.orig/arch/arm/mach-omap2/powerdomains44xx.h	2011-01-06 13:14:35.000000000 +0530
+++ android-2.6.35/arch/arm/mach-omap2/powerdomains44xx.h	2011-01-25 11:31:48.507842195 +0530
@@ -96,7 +96,7 @@
 	.pwrsts		  = PWRSTS_OFF_RET_ON,
 	.pwrsts_logic_ret = PWRDM_POWER_OFF,
 	.banks		  = 2,
-	.context_offset	  = 0x24,
+	.context_offset	  = 0x2c,
 	.pwrsts_mem_ret	= {
 		[0] = PWRDM_POWER_RET,	/* aessmem */
 		[1] = PWRDM_POWER_OFF,	/* periphmem */
